interview-questions.md

```diff
@@ -7,60 +7,66 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is git? What is the difference between git and Github?
 
-  Your answer:
+  Your answer: git is a command or declaration on the terminal to inform the terminal to grab or create a file in the Github repository, desktop folder, or another designated folder or drive. Github is a wonderful storage center for files. It is the space that the "git" command collects and places files. Github allows several people to work on a project and allows those people to create temporary folders to test out their coding before placing into the main application of the project.
```

Comment:
Honestly this kind of answer will get a job super fast. I really appreciate how you got into the details in your research portion. Phenomenal work